docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+    +------------+-------------------------------------------------------------------------------------+
+    | Field      | Value                                                                               |
+    +------------+-------------------------------------------------------------------------------------+
+    | created_at | 2023-05-10T13:31:46Z                                                                |
+    | file       | /v2/images/1a50720b-af1c-4a16-ba19-b89dbee251f0/file                                |
+    | id         | 1a50720b-af1c-4a16-ba19-b89dbee251f0                                                |
+    | min_disk   | 50                                                                                  |
+    | min_ram    | 0                                                                                   |
+    | name       | nanisivik_snap                                                                      |
+    | owner      | 94109c764a754e24ac0f6b01aef82359                                                    |
+    | properties | base_image_ref='d6b012ee-c6d3-4672-9399-b87d025ddb14', boot_roles='load-            |
+    |            | balancer_member,_member_,swiftoperator,creator', hw_machine_type='pc',              |
+    |            | hw_qemu_guest_agent='yes', image_type='snapshot',                                   |
+    |            | instance_uuid='19ffeb4d-3016-481f-886e-6caecc525fc2', locations='[]',               |
+    |            | os_hidden='False', owner_project_name='Meanwhile-in-Drama',                         |
+    |            | owner_user_name='kolderson', user_id='c096cf99f65a4d22a6954b67d2ec11d7'             |
+    | protected  | False                                                                               |
+    | schema     | /v2/schemas/image                                                                   |
+    | status     | queued                                                                              |
+    | tags       |                                                                                     |
+    | updated_at | 2023-05-10T13:31:46Z                                                                |
+    | visibility | private                                                                             |
+    +------------+-------------------------------------------------------------------------------------+
+    ```
+
+    At first, the status of the snapshot is `queued`. To make sure the
+    snapshot has been successfully created, type:
+
+    ```bash
+    openstack image show nanisivik_snap -c status -f value
+    ```
+
+    You should see this:
+
+    ```plain
+    active
+    ```
+
+## Creating a volume from the snapshot
+
+=== "{{gui}}"
+    After a few seconds, the snapshot will be created and listed in the
+    *Snapshots* tab. Before you proceed, take note of its size. At the
+    right-hand side of the snapshot row, click the :fontawesome-database:
```

Comment:
:fontawesome-database: is not one of [shortcodes](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/) that mkdocs-material supports. Did you mean to use `:fontawesome-solid-database:`, perhaps?